lib/visitor/strategy/default_strategy.dart

```diff
@@ -136,6 +136,8 @@ class DefaultStrategy implements IVisitor {
         // Update the state when panic mode is ON
         visitor.flagshipDelegate.onUpdateState(state);
       } else {
+        state = Status.READY;
+        visitor.flagshipDelegate.onUpdateState(state);
```

Comment:
Why not move that line outside the if/else, since you're calling the exact same method and parameter on both the if and the else

Reply:
ohh yess right